lib/webenginesettings.cpp

```diff
@@ -110,17 +127,32 @@ void SailfishOS::WebEngineSettings::initialize()
     engineSettings->setPreference(QStringLiteral("intl.accept_languages"),
                                   QVariant::fromValue<QString>(langs));
 
+    // Ensure the renderer is configured correctly
+    engineSettings->setPreference(QStringLiteral("gfx.webrender.force-disabled"),
+                                  QVariant(true));
```

Comment:
This could go to embedding.js to the gecko side.

Reply:
I've moved this out of here too.